packages/gestalt/src/Masonry.js

```diff
@@ -377,13 +386,23 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
     this.forceUpdate();
   }
 
+  renderItem(item: {| +data: T, +itemIdx: number, +isMeasuring: boolean |}): Node {
+    const { Item, renderItem } = this.props;
+    if (renderItem) {
+      return renderItem(item);
+    }
+    if (Item) {
+      return <Item data={item.data} itemIdx={item.itemIdx} isMeasuring={item.isMeasuring} />;
+    }
+    return null;
```

Comment:
This should probably throw an error if we get here?

Reply:
addressed in https://github.com/pinterest/gestalt/pull/2616/commits/74257966b14a3574ea6e3c7e7073f10fff98f930

Reply:
that was bad. fixed again here: https://github.com/pinterest/gestalt/pull/2616/commits/88c61479059b0b13a29977170a0413cc45c773be